iml-wire-types/src/snapshot.rs

```diff
@@ -34,3 +34,23 @@ pub struct Destroy {
     #[cfg_attr(feature = "cli", structopt(short = "f", long = "force"))]
     pub force: bool,
 }
+
+#[derive(serde::Deserialize, Debug)]
+#[cfg_attr(feature = "cli", derive(StructOpt))]
+/// Ask agent to mount the snapshot
+pub struct Mount {
+    /// Filesystem name
+    pub fsname: String,
+    /// Snapshot name
+    pub name: String,
+}
+
+#[derive(serde::Deserialize, Debug)]
+#[cfg_attr(feature = "cli", derive(StructOpt))]
+/// Ask agent to unmount the snapshot
+pub struct Unmount {
```

Comment:
Btw, I was already thinking that simple tuple was good :) At least the same structure could be used for both mounting and unmounting.

Reply:
Tuple is good, but would need to be a tuple struct. The value here is being able to have the caller break at compile time if this changes.